src/theory/quantifiers/ceg_instantiator.h

```diff
@@ -41,14 +41,24 @@ class CegqiOutput {
 class Instantiator;
 
 
+class TermProperties {
```

Comment:
A comment describing the purpose of this class would be nice.

Reply:
See general comment [1] below.